Source/Fuse.Controls.ScrollView/ScrollView.Layout.uno

```diff
@@ -221,6 +221,26 @@ namespace Fuse.Controls
 				DistanceToViewLinear(min.Y, max.Y, ScrollPosition.Y, ActualSize.Y));
 		}
 		
+		/**
+			The distance from the visible view area start for the provided rectangle. It will be zero if any part of it resides before the view area start.
+		*/
+		internal float2 DistanceFromViewStart(float2 min, float2 max)
+		{
+			return float2(
+				DistanceFromViewStartLinear(min.X, max.X, ScrollPosition.X, ActualSize.X),
+				DistanceFromViewStartLinear(min.Y, max.Y, ScrollPosition.Y, ActualSize.Y));
+		}
+		
+		/**
+			The distance from the visible view area end for the provided rectangle. It will be zero if any part of it resides after the view area end.
+		*/
+		internal float2 DistanceFromViewEnd(float2 min, float2 max)
```

Comment:
This pattern of functions seems to be proliferating. Since only WhileVisible... uses them it might make more sense to have just one generic measuring function and let WhileVisible... impement the logic parts.